habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -134,6 +137,14 @@ class StopActionConfig(ActionConfig):
     type: str = "StopAction"
 
 
+@attr.s(auto_attribs=True, slots=True)
+class VelocityStopActionConfig(ActionConfig):
+    r"""
+    In Navigation tasks only, the velocity stop action is an action that takes continuous space input.
```

Comment:
what does `an action that takes continuous space input` mean ?

Reply:
Modified the text